client/src/components/miniComponents/ProfileSideMenu.jsx

```diff
@@ -0,0 +1,14 @@
+import { Button } from "bootstrap";
+import "../../assets/css/ProfileSideMenu.css";
+import { useState } from "react";
+
+export default function SideMenu(props) {
```

Comment:
Please either change the file name to SideMenu or the function name to ProfileSideMenu.